force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -259,11 +297,11 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Integer performDivisionAndCast(Double a, Double b) {
         // Perform the division and cast (round down) off the result.
-        Double divisionResult = null;
+        Double divisionResult = a / b;
 
         // Write the code for type casting the divisionResult to an Integer
-        Integer roundedResult = null;
-        return null; // Replace null with the variable you used to store the result
+        Integer roundedResult = divisionResult.intValue();
```

Comment:
Nicely done! You can also just cast to an integer like this:
`Integer roundedResult = (Integer) divisionResult;`